CHANGES.md

```diff
@@ -0,0 +1,9 @@
+# moray Changelog
+
+## 2.4.0
```

Comment:
I'm not really sure what to do with the versioning here between `mantav1` and `master`/`manatav2`? I don't think it matter a whole lot, but happy for feedback on it.

Reply:
I think it is fine:
1. I don't believe the package.json versioning is being *used* by anything.
2. The "master" and "mantav1" branches of moray will be the same after this backport, with the exception of the recent TRITON-2065 min platform change.

I'd say that if the "master" branch adds a change that breaks compat with the "mantav1" branch that it should probably bump its major version (to 3.x) and leave 2.x for any subsequent "mantav1" changes -- to the degree that it matters give bullet #1 above.